### PR TITLE
Modify containers, JeOS, SLEM and WSL toml files to monitor newer versions

### PR DESCRIPTION
**Pull request by pablo-herranz**
Changes in each .toml file:

- Containers:
  - Replaced 15-SP4 with 15-SP5 in BCI Base Images
  - Move 15-SP4 to maint. updates

- JeOS:
  - Add 15-SP4 to maint. updates

- SLEM:
  - Replaced 5.3 with 5.4 in Latest Images
  - Add 5.4 to Latest Images
  - Add 5.2 in Public Cloud
  - Remove 5.1 in Public Cloud
  - Add SLEM for Rancher
  - Add 5.3 to maint. updates
  - Remove 5.0 from maint. updates

- WSL
  - Removed 15-SP2
  - Added 15-SP5
  - Add monitoring for Windows installations

### CODE REVIEW

File: _review/qec-sle-wsl.toml [hunk 1]
```diff
@@ -3,13 +3,18 @@
 Instance = "https://openqa.suse.de"              # openQA instance to query
 RabbitMQ = "amqps://suse:suse@rabbit.suse.de"    # RabbitMQ instance to query
 RabbitMQTopic = "suse.openqa.job.done"           # RabbitMQ topic to query
-HideStatus = [ "scheduled", "passed", "softfailed", "running", "reviewed" ]           # Hide scheduled and passed jobs
+HideStatus = [ "scheduled", "passed", "softfailed", "running", "reviewed" ] # Hide scheduled and passed jobs
```
Comment (m-dati) [inline _review/qec-sle-wsl.toml]: If you have time, not mandatory, update the comment too, i.e.: `Hide jobs in a specific status`

Comment (pablo-herranz) [reply _review/qec-sle-wsl.toml]: Changed :)

**Comment by pdostal** [comment]
Please provide less generic commit message and pull request title (e.g. specify which files you changing and how).

This pull request continues from #99.

**Review by pdostal — APPROVED** [review]
🚀

**Comment by grisu48** [comment]
image

Can you please sign the commit? You can do this by add the `signingKey` to your `~/.gitconfig` and amend the commit, e.g.

```
[user]
    name = ...
    email = ...
    signingKey = ...
```

File: _review/qec-containers.toml [hunk 2]
```diff
@@ -42,6 +46,11 @@ MaxLifetime = 86400
 
 
 ## Maintenance updates
+[[Groups]]
+Name = "Containers Maintenance Updates 15-SP4"
+Params = { groupid = "417", build = "%yesterday%-1", version="15-SP4" }
+MaxLifetime = 86400
```
Comment (grisu48) [inline _review/qec-containers.toml:53]: Not needed with this `build` variable.

File: _review/qec-jeos.toml [hunk 3]
```diff
@@ -16,6 +16,10 @@ Params = { groupid = "131" }
 MaxLifetime = 86400
 
 ## Maintenance updates
+[[Groups]]
+Name = "JeOS Maintenance Updates 15-SP4"
+Params = { groupid = "419", build = "%yesterday%-1", version = "15-SP4" }
+MaxLifetime = 86400
```
Comment (grisu48) [inline _review/qec-jeos.toml]: Not needed with this `build` variable

File: _review/qec-sle-micro.toml [hunk 4]
```diff
@@ -3,34 +3,54 @@
...
 ## Latest SLE Micro images
 [[Groups]]
-Name = "Lastest SLE Micro (5.3)"
+Name = "Latest SLE Micro (5.4)"
 Params = { groupid = "410" }
 MaxLifetime = 86400
 
 [[Groups]]
-Name = "SLEM 5.2 on Publiccloud"
+Name = "SLEM 5.1 on Public Cloud"
```
Comment (grisu48) [inline _review/qec-sle-micro.toml]: I know that I did it wrong in the beginning but still .... Mind reversing the order to ascending? (First 5.4, then 5.3, 5.2, ...) It looks a bit weird to have first 5.4 then 5.1 then 5.2 ....

File: _review/qec-sle-micro.toml [hunk 5]
```diff
@@ -3,34 +3,54 @@
...
+[[Groups]]
+Name = "SLE Micro for Rancher"
+Params = { groupid = "449" }
+MaxLifetime = 86400
+
 ## Maintenance updates
+[[Groups]]
+Name = "SLE Micro Maintenance Updates 5.3"
+Params = { groupid = "420", build = "%yesterday%-1", version = "5.3" }
+MaxLifetime = 86400
```
Comment (grisu48) [inline _review/qec-sle-micro.toml]: Not needed with this `build` variable

File: _review/qec-sle-wsl.toml [hunk 6]
```diff
@@ -20,13 +25,14 @@ Name = "SLE 15-SP3 WSL"
...
-Name = "SLE 12-SP5 WSL"
-Params = { groupid = "288" }
+Name = "Windows 11 installation"
+Params = { version="11", groupid="287" }
 MaxLifetime = 86400
 
+[[Groups]]
+Name = "Windows 10 installation"
+Params = { version="10", groupid="287" }
+MaxLifetime = 86400
```
Comment (grisu48) [inline _review/qec-sle-wsl.toml]: Please add a newline at the end.

**Review by grisu48 — CHANGES_REQUESTED** [review]
Thanks Pablo, this looks already great! Some minor changes are required but then let's go! :rock:

File: _review/qec-containers.toml (hunk 2 above)
Comment (pablo-herranz) [reply _review/qec-containers.toml:52]: Not needed in any of the versions or just 15-SP4?

File: _review/qec-sle-micro.toml (hunk 4 above)
Comment (pablo-herranz) [reply _review/qec-sle-micro.toml]: Yes, it is weird to me too, but it's how it shows up in OSD menu, that's why I choose it this way. Want me to reverse it anyway?

image

Comment (grisu48) [reply _review/qec-sle-micro.toml]: Yes please, it looks weird. I do not see why we should follow weirdness on OSD.

**Comment by grisu48** [comment]
Is this PR still active?

**Comment by pablo-herranz** [comment]
Yup, nobody merged it :(

**Comment by grisu48** [comment]
> Yup, nobody merged it :(

Oh sorry, I only saw the unresolved discussions and assumed this is till work in progress.

If you rebase it and poke me, I can merge it.

**Review by grisu48 — APPROVED** [review]
LGTM!

**Pull request merged**